Samples/Samples.Dialogs/appsettings.Development.json

```diff
@@ -13,5 +13,6 @@
     "ClientId": "",
     "ClientSecret": ""
   },
-  "BotEndpoint": ""
+  "BotEndpoint": "",
+  "USE_MINIMAL_API": false
```

Comment:
dont think this is the place, but if we decided to keep it, move the value to the main appSettings.json